torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -4,43 +4,42 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 namespace {
-  py::object module_;
-  py::object runUDFFunction_;
-  py::object loadResultFunction_;
+py::object runUDFFunction_ = py::none();
+py::object loadResultFunction_ = py::none();
 } // anonymous namespace
 
 namespace PythonRpcHandler {
-  void init() {
-    AutoGIL ag;
-    if (module_ == nullptr) {
-      module_ = py::module::import("torch.distributed.internal_rpc_utils");
-    }
-    if (runUDFFunction_ == nullptr) {
-      runUDFFunction_ = module_.attr("run_python_udf_internal");
-    }
-    if (loadResultFunction_ == nullptr) {
-      loadResultFunction_ = module_.attr("load_python_udf_result_internal");
-    }
+void init() {
+  AutoGIL ag;
+  py::object module =
+      py::module::import("torch.distributed.internal_rpc_utils");
+  if (module.is(py::none())) {
```

Comment:
When this can be none? And why import it again would work?

Reply:
also address the comment

Reply:
Will remove this. import failure means wrong code.